epidose/device/beacon_tx_unlinkable_d.py

```diff
@@ -84,6 +85,17 @@ def set_transmit(interface, ephid, rssi):
     run_command(cmd)
 
 
+def generate_random_bdaddr():
+    bdaddr = secrets.token_hex(6)
+    bdaddr = list(bdaddr)
+    locally_administer_elements = ['2', '6', 'a', 'e']
+    
+    # The above values denote that the bdaddr is locally administared
+    bdaddr[1] = secrets.choice(locally_administer_elements)
```

Comment:
Please comment each line with an example of what bdaddr will contain.

Reply:
I have added an example and comments to each line of this function.